<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>789</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-image: url(&quot;C:/Users/DANALAP/Downloads/tai-hinh-nen-xe-hoi-cho-may-tinh-15.jpg&quot;);
    background-repeat: no-repeat;
    background-position: center;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStatesMinorOutlyingIslands"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #fff;
}
</string>
   </property>
   <widget class="QGroupBox" name="groupLogin">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>841</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#groupLogin {
    background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.306818 rgb(246,226,253), stop:0.886364 rgb(219,237,252));

    border: none;
    border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="txtPassword">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>150</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border-radius: 20px;
	padding-left: 12px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtUsername">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border-radius: 20px;
color: #371881;
	padding-left: 12px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogin">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>230</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btnLogin:hover:!pressed {
    
	border-radius: 10px;
}
QPushButton#btnLogin {
   background-color: #743de3;
   border : none;
   color: white;
   border-radius: 10px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.306818 rgb(131,56,236), stop:0.886364 #3a86ff);
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #371881;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.306818 rgb(246,226,253), stop:0.886364 rgb(219,237,252));
}</string>
     </property>
     <property name="text">
      <string>Username :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #371881;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.306818 rgb(246,226,253), stop:0.886364 rgb(219,237,252));
}</string>
     </property>
     <property name="text">
      <string>Password :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>500</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(194, 188, 191);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>500</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(194, 188, 191);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.306818 rgba(143, 0, 146, 255), stop:0.886364 rgba(0, 100, 245, 255));
   background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.306818 rgb(246,226,253), stop:0.886364 rgb(219,237,252));
}</string>
     </property>
     <property name="text">
      <string>CAR LISENCE PLATE RECOGNITION APPLICATION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRegister">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>230</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton#btnRegister {
   background-color: #27a645;
   border : none;
   color: white;
   border-radius: 10px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0.306818 rgb(131,56,236), stop:0.886364 #3a86ff);
}</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-240</x>
      <y>-100</y>
      <width>1551</width>
      <height>1281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>2</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imageUI/Audi-R8-HD-Image.jpg</pixmap>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>groupLogin</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
